planet/cli/utils.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Planet Labs, PBC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for CLI modules"""
+import asyncio
+from functools import wraps
+import json
+
+import click
+
+import planet
+
+
+# https://github.com/pallets/click/issues/85#issuecomment-503464628
+def coro(f):
+    '''Wraps async functions so they can be run sync with Click.'''
+    @wraps(f)
+    def wrapper(*args, **kwargs):
+        return asyncio.run(f(*args, **kwargs))
+    return wrapper
+
+
+def handle_exceptions(f):
```

Comment:
"handle exceptions" is a little bit too vague to be a good name.

```suggestion
def translate_exceptions(func):
    """Translate internal exceptions to ClickException."""
```

Reply:
And sometimes single char variable names get in the way of debugging. Maybe we could limit use to `i` for looping or `f` for file objects, but not in function declarations?

Reply:
This is an excellent change and docstring, thank you for providing this suggestion! And I'm with you on the single char variable name, good point.

Reply:
a note on the `f` var. I was just familiarizing myself with @wraps and noticed that the official Python documentation uses `f` as the argument name for the example usage of `wraps` ([ref]:(https://docs.python.org/3/library/functools.html#functools.wraps))

```python
>>> from functools import wraps
>>> def my_decorator(f):
...     @wraps(f)
...     def wrapper(*args, **kwds):
...         print('Calling decorated function')
...         return f(*args, **kwds)
...     return wrapper
```